tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```

Comment:
Any reason for this change?
When you run `yarn test` the CRA test config will change it back to `react-jsx`. From the CI test log:

```
Run yarn test
yarn run v1.22.5
$ react-scripts test
The following changes are being made to your tsconfig.json file:
  - compilerOptions.jsx must be react-jsx (to support the new JSX transform in React 17)
```

Reply:
I will check more into this, for some reason my vscode was saying that react-jsx isn't a valid option. Then I changed and everything felt right

Reply:
https://stackoverflow.com/questions/64974648/problem-with-visual-studio-code-using-react-jsx-as-jsx-value-with-create-react

Reply:
Seems like your VSCode is using an old version of Typescript like they suggested, I didn't get that error, I'm not sure if it's my VSCode version or configuration. Are you using the latest version of VSCode?

Reply:
The [latest version](https://code.visualstudio.com/updates/v1_52#_typescript-41) that was released in November added Typescript v4.1.2, which is basically the same one that CRA uses (v4.1.3), so that should work I think